local/container.go

```diff
@@ -105,33 +108,41 @@ func (c *container) Items(prefix, cursor string, count int) ([]stow.Item, string
 		if f.IsDir() {
 			continue
 		}
-		path, err := filepath.Abs(filepath.Join(c.path, f.Name()))
-		if err != nil {
-			return nil, "", err
-		}
 		if !strings.HasPrefix(f.Name(), prefix) {
 			continue
 		}
 		item := &item{
-			path: path,
+			relPath:   f.Name(),
+			container: c,
 		}
 		items = append(items, item)
 	}
 	return items, cursor, nil
 }
 
 func (c *container) Item(id string) (stow.Item, error) {
-	path := id
-	info, err := os.Stat(path)
+	var fullPath string
+	if filepath.IsAbs(id) {
+		//Absolute path - id has native OS semantics
+		fullPath = id
+	} else {
+		//relative path - id has '/' semantics
+		fullPath = filepath.Join(c.path, filepath.FromSlash(id))
+	}
+	info, err := os.Stat(fullPath)
```

Comment:
the err could be something other than `os.IsNotExist`, and we will want to check on that as well